wire-runtime/src/commonMain/kotlin/com/squareup/wire/ProtoReader.kt

```diff
@@ -447,6 +447,21 @@ class ProtoReader(private val source: BufferedSource) {
     (protoAdapter as ProtoAdapter<Any>).encodeWithTag(unknownFieldsWriter, tag, value)
   }
 
+  /**
+   * Returns the min length of the next field in bytes. Some encodings have a fixed length, while others
+   * have a variable length. LENGTH_DELIMITED fields have a known variable length, while VARINT fields
+   * could be as small as a single byte.
+   */
+  fun nextFieldLengthInBytes(): Long {
+    return when (nextFieldEncoding) {
+      FieldEncoding.LENGTH_DELIMITED -> limit - pos
+      FieldEncoding.FIXED32 -> 4
+      FieldEncoding.FIXED64 -> 8
+      FieldEncoding.VARINT -> 1
```

Comment:
It could be as little as 1, but it might be more? The var in varint is variable-length

Fix by renaming the function to contain say, _minimum_ or something?

Reply:
Yep you're right, I have it in the comment and then totally forgot to update the name